charts/tekton-pipelines/templates/deployment-tekton-pipelines-remote-resolvers.yaml

```diff
@@ -64,13 +64,7 @@ spec:
       containers:
         - name: controller
           image: {{ .Values.remoteresolver.deployment.image }}
-          resources:
-            requests:
-              cpu: 100m
-              memory: 100Mi
-            limits:
-              cpu: 1000m
-              memory: 2Gi
+          resources: {{- toYaml .Values.controller.resources | nindent 12 }}
```

Comment:
Was it intended to have the `remote-resolver` use the same resources as the `controller`?
`{{- toYaml .Values.controller.resources | nindent 12 }}`

There is also a .Values.remoteresolver.resources